microservices/configcheck/src/main/java/datawave/microservice/configcheck/CommandRunner.java

```diff
@@ -0,0 +1,225 @@
+                        }
+                    } catch (Exception e) {
+                        log.error("Encountered exception during analyze", e);
+                    } finally {
+                        if (output == null || output.isEmpty()) {
+                            output = "USAGE \n" + "configcheck " + ANALYZE + " [FILE] [--" + FULL_REPORT + "] [--" + CONFIGDIR + "=[PATH]] --" + PROPERTIES
+                                            + "=[FILE...] [--" + OUTPUT + "=[FILE]]\n" + "configcheck " + ANALYZE + " [FILE] [--" + FULL_REPORT + "] [--"
+                                            + CONFIGDIR + "=[PATH]] --" + YAML + "=[FILE...] [--" + OUTPUT + "=[FILE]]\n\n" + "NOTE\n"
+                                            + "  Only XML files can be analyzed.\n\n" + "OPTIONS\n" + "  --" + FULL_REPORT + "\n"
+                                            + "    Generate a full report containing a mapping of keys to placeholders, values, and other useful "
+                                            + "      information.\n\n" + "  --" + CONFIGDIR + "=[PATH]\n"
+                                            + "    The directory containing your configuration files\n\n" + "  --" + PROPERTIES + "=[FILE...]\n"
+                                            + "    A comma-separated list of properties files to load.  If " + CONFIGDIR + " is specified, \n"
+                                            + "      the files will be loaded relative to that path.  Configuration files are loaded in order, so \n"
+                                            + "      properties in subsequent files will override prior files.\n\n" + "  --" + YAML + "=[FILE...]\n"
+                                            + "    A comma-separated list of yaml files to load.  If " + CONFIGDIR + " is specified, \n"
+                                            + "      the files will be loaded relative to that path.  Configuration files are loaded in order, so \n"
+                                            + "      properties in subsequent files will override prior files.\n\n" + "  --" + OUTPUT + "=[FILE]\n"
+                                            + "    The file where the output should be written.\n";
+                        }
+                    }
+                    break;
+                case COMPARE:
+                    try {
+                        if (!help) {
+                            output = runCompareCommand(args);
+                        }
+                    } catch (Exception e) {
+                        log.error("Encountered exception during compare", e);
+                    } finally {
+                        if (output == null || output.isEmpty()) {
+                            output = "USAGE \n" + "configcheck " + COMPARE + " [FILE] [FILE] [--" + OUTPUT + "=[FILE]]\n\n" + "NOTE\n"
+                                            + "  Compares the key/values generated by the " + ANALYZE + " command.\n\n" + "OPTIONS\n" + "  --" + OUTPUT
```

Comment:
specify that this cannot be the "--fullReport" analysis output.  That does not work with the compare command.

Reply:
I updated the code so that it CAN be a full report.